packages/bot-skeleton/src/scratch/blocks/Binary/Trade Definition/trade_definition_market.js

```diff
@@ -61,9 +62,33 @@ Blockly.Blocks.trade_definition_market = {
         // market options and Jump Diffusion Indices from
         // submarket options for Synthetic Indices
         // until multipliers are available for DBot
-        const market_options = active_symbols
-            .getMarketDropdownOptions()
-            .filter(option => option[1] !== 'cryptocurrency');
+        const excludedCountries = [
+            'gb',
+            'ie',
+            'pt',
+            'se',
+            'pl',
+            'cy',
+            'nl',
+            'si',
+            'lv',
+            'hu',
+            'cz',
+            'ro',
+            'dk',
+            'fi',
+            'lt',
+            'hr',
+            'ee',
+            'sk',
+            'bg',
+            'at',
+        ];
+        const { is_virtual, residence } = DBotStore.instance.client;
+        let market_options = active_symbols.getMarketDropdownOptions().filter(option => option[1] !== 'cryptocurrency');
+        if (is_virtual && excludedCountries.includes(residence)) {
+            market_options = market_options.filter(option => option[1] !== 'forex');
+        }
```

Comment:
maybe it's better to set a const like that and move up it to all const declarations
```suggestion
        const isForexAvailable = is_virtual && excludedCountries.includes(residence)
```

Reply:
it will be better `isForexExcluded`